spec/service_consumers/pact_helper.rb

```diff
@@ -26,6 +26,9 @@
   # honours_pact_with 'HCA Post' do
   #   pact_uri 'tmp/hca-va.gov_api.json'
   # end
+  honours_pact_with 'Forms' do
+    pact_uri 'tmp/forms.gov_api.json'
```

Comment:
why was this pact put in `tmp` instead of being checked in?

Reply:
Ah yes. Because I have no plans to merge this PR. If I were going to merge this code, I'd publish the PACT JSON to the PACT broker service (currently hosted on Heroku), not check in a JSON file. I put the code for the tmp file in the PR description, in case anyone else wanted to test locally (without publishing to the PACT broker). Github won't let me attach a JSON file to the description, or I'd have done that.